rpmlint/descriptions/FilesCheck.toml

```diff
@@ -409,3 +409,11 @@ A file in /usr/bin is a link to a script in a different place with a shebang.
 rpm won't be able to inject the needed interpreter as dependency, so it should
 be done manually.
 """
+
```

Comment:
we should work on the guidance here. for all non %ghost entries it is often better to fix things in %install e.g.

only for %ghost we would need %attr() in %files. and then we should point the user to systemd tmpfiles.d (we could maybe even check if the file is covered by tmpfiles.d stuff and then spit out the correct %ghost %attr() ... line based on what we see in tmpfiles.d


if that comfort step is not possible we should mention that attr should match what is in the tmpfiles.d

Reply:
What do you think should be the best way to handle this? To improve the error message or to improve the check and show a different warning message depending on if the file is ghost.

Reply:
well if we have access to the tmpfiles.d files I would definitely prefer to have the code spit out the correct stuff to give our packagers more guidance. do we have access to unpacked files at the point?

Reply:
otherwise the description should at least give a hint like:

`please ensure that the file mode (directory/file/symlink), ownership and permissions match what will be created on the filesystem later`

Reply:
> well if we have access to the tmpfiles.d files I would definitely prefer to have the code spit out the correct stuff to give our packagers more guidance. do we have access to unpacked files at the point?

During this check we've access to all files defined in the rpm package. The package is expanded in the temporal folder, and we read those files to check for shebang, for example: https://github.com/rpm-software-management/rpmlint/blob/main/rpmlint/checks/FilesCheck.py#L879-L880

Reply:
Help me to understand this better, do you know about some example pkg that we can use in a test case?

Reply:
redis e.g.

Reply:
sysconfig with its sysconfig-netconfig subpackage is also a good candidate probably.